JS/Mery/Health-App/src/examples/FitnessApp.jsx

```diff
@@ -0,0 +1,168 @@
+                  Plan list
+                </H4>
+                <IconButton
+                  size='small'
+                  rounded
+                  text={false}
+                  color='var(--primary)'
+                  dark={dark}
+                >
+                  <Icon path={mdiPlus} size={0.8} />
+                </IconButton>
+              </div>
+            )}
+            selectedIndex={1}
+            dark={dark}
+          >
+            <PlanScreen dark={dark} />
+          </FitnessAppScreen>
+          <FitnessAppScreen
+            title='Food'
+            selectedIndex={2}
+            dark={dark}
+          >
+            <FoodScreen />
+          </FitnessAppScreen>
+          <FitnessAppScreen
+            title='Notifications'
+            selectedIndex={3}
+            dark={dark}
+          >
+            <ListItem
+              style={{
+                marginTop: '10px',
+              }}
+              link
+              active
+              rounded
+              title={'Notification 1'}
+              subtitle={
+                'Do not forget to drink water'
+              }
+              
+            />
+            <ListItem
+              style={{
+                marginTop: '10px',
+              }}
+              link
+              active
+              rounded
+              title={'Notification 2'}
+              subtitle={
+                'Please do exercises'
+              }
+              
+            />
+            <ListItem
+              style={{
+                marginTop: '10px',
+              }}
+              link
+              active
+              rounded
+              title={'Notification 2'}
+              subtitle={
+                'Please do exercises'
+              }
+              
+            />
+            <ListItem
+              style={{
+                marginTop: '10px',
+              }}
+              link
+              active
+              rounded
+              title={'Notification 3'}
+              subtitle={
+                'Running goal is not completed'
+              }
+              
+            />
+          </FitnessAppScreen>
+          <FitnessAppScreen
+            title='Account'
+            selectedIndex={4}
+            dark={dark}
+          >
+            <Card
+              dark={dark}
+              elevation={3}
+              style={{
+                display: 'flex',
+                marginTop: '10px',
+                marginBottom: '20px',
+                marginLeft: '80px',
+                width: '90px',
+                height: '90px',
+                borderRadius: '30px',
+                alignItems: 'center',
+                justifyContent: 'center'
+              }}
+            >
+              <Card flat>
+                <Icon path={mdiAccount} size={2.5} color='var(--primary)' />
+              </Card>
+            </Card>
+            <Subtitle2>Gender: Female</Subtitle2>
```

Comment:
Why without space and with spaces bellow

Reply:
Resolved